index.html

```diff
@@ -889,6 +884,76 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
+    <section id='external-content'>
+      <h3>External LDP-NR Content</h3>
+      <p class='informative'>
+        Variability among client types and locations may mean that LDP-NR
+        content is addressed in ways that are external to LDP-S but not resolvable
```

Comment:
I am guessing `LDP-S` is intended to mean `LDP Server`... which should probably be `the Fedora server`...as `the server` is probably not exactly accurate in this situation.